client/src/components/AttributeTrackFilters.vue

```diff
@@ -0,0 +1,466 @@
+                label="Attribute"
+                class="mx-2"
+              />
+            </v-row>
+            <v-row dense>
+              <v-switch
+                v-model="editingFilterEnabled"
+                label="Enabled"
+                class="mx-2"
+              />
+              <v-tooltip
+                open-delay="100"
+                bottom
+              >
+                <template #activator="{ on }">
+                  <v-icon v-on="on">
+                    mdi-information
+                  </v-icon>
+                </template>
+                <span
+                  class="ma-0 pa-1"
+                >
+                  Will configure the filter to be enabled when opening the dataset.
+                </span>
+              </v-tooltip>
+
+              <v-switch
+                v-model="editingPrimaryDisplay"
+                label="Primary Display"
+                class="mx-2"
+              />
+              <v-tooltip
+                open-delay="100"
+                bottom
+              >
+                <template #activator="{ on }">
+                  <v-icon v-on="on">
+                    mdi-information
+                  </v-icon>
+                </template>
+                <span
+                  class="ma-0 pa-1"
+                >
+                  The Track Attribute Filter will be displayed on the main panel below the type list.
+                </span>
+              </v-tooltip>
+
+              <v-switch
+                v-model="editingUserDefined"
+                label="User Editable"
+                class="mx-2"
+              />
+              <v-tooltip
+                open-delay="100"
+                bottom
+              >
+                <template #activator="{ on }">
+                  <v-icon v-on="on">
+                    mdi-information
+                  </v-icon>
+                </template>
+                <span
+                  class="ma-0 pa-1"
+                >
+                  The values for the filter can be edited by the user.
+                </span>
+              </v-tooltip>
+            </v-row>
+            <v-row dense>
+              <v-select
+                v-model="editingAtrOp"
+                :items="editingOps"
+                label="Operator"
+                :rules="[v => !!v || 'Operator is required']"
+                required
+                @change="changeAttributeType"
+              />
+            </v-row>
+            <v-row dense>
+              <v-text-field
+                v-if="!['range', 'in'].includes(editingAtrOp)"
+                v-model="editingAtrVal"
+                :type="attributeTypes[editingAtrKey] === 'text' ? 'text' : 'number'"
+                :label="editingAtrOp === 'rangeFilter' ? 'Default Value' : 'Test/Default Value'"
+                persistent-hint
+                :hint="editingAtrOp === 'rangeFilter' ? 'Default Value' : 'Test/Default Value'"
+              />
```

Comment:
If type is a number it should have the same step value as it does in the side panel '0.01'